pydhi/__init__.py

```diff
@@ -1,3 +1,13 @@
+from pydhi import dfs_util as dfs_util
+from pydhi import dfs3 as dfs3
+from pydhi import dfs2 as dfs2
+from pydhi import dfs0 as dfs0
+import platform
+from System.Runtime.InteropServices import GCHandle, GCHandleType
+from DHI.Generic.MikeZero.DFS.dfs123 import *
```

Comment:
Would it make sense to specify which things are imported here, instead of using the * imports?

Reply:
so much sense.

Reply:
never `import *`. You can't lint your code anymore, don't know where objects in the namespace come from, and there can be nasty shadowing.

Reply:
But I think fixing that is up to the author, please. @marcridler